.eslintignore

```diff
@@ -13,3 +13,4 @@ https-dev-config/localhost.key
 Dockerfile
 elk-translation-status.json
 docs/translation-status.json
+telemetry/generated
```

Comment:
adding the generated files to the eslint ignore so they don't constantly get overridden